docker/omnibus-centos6/Dockerfile

```diff
@@ -58,4 +57,18 @@ RUN chown jenkins /var/cache/omnibus
 RUN chown jenkins /opt/metasploit-framework
 RUN chown -R jenkins /opt/chef/embedded/lib/ruby/gems
 
+RUN echo "#!/usr/bin/expect -f" > /usr/bin/signrpm && \
```

Comment:
Heh, yeah - this crazy shell-in-DSL stuff makes me nervous about mixing too many paradigms at once. :)